charts/gp-reporting-kafka-health/v0.0.1/questions.yml

```diff
@@ -0,0 +1,162 @@
+questions:
+  # General environment settings
+  - variable: environment.parentName
+    type: enum
+    required: true
+    group: "Environment"
+    label: Environment Parent
+    description: The name of the parent environment we will deploy into and funcation as.
+    options:
+      - sandbox
+      - canary
+      - production
+    default: sandbox
+
+  - variable: environment.modifierValue
+    type: enum
+    required: false
+    group: "Environment"
+    label: Environment Modifier
+    description: The modifier name for the environment we will deploy into and funcation as.
+    options:
+      - prime
+      - qa
+      - testing
+      - 01
+      - 02
+      - 03
+      - 04
+      - 05
+      - 06
+      - 07
+      - 08
+      - 09
+      - 10
+    default: "01"
+
+  ####
+  #### Reporting Kafka Health Service Settings
+  ####
+  - variable: build.commitSHA
+    type: string
+    required: true
+    group: "Application"
+    label: Application Build SHA
+    description: The commit SHA of the GitPrime Application build you want to deploy.
+    default: latest
+
+  - variable: volumes.enableLocalMount
+    type: boolean
+    required: true
+    group: "Application"
+    label: "Use Local Mount"
+    description: "If enabled, use the node's local storage directory for repository storage"
+    default: false
+    show_subquestion_if: "true"
+    subquestions:
+      - variable: volumes.nodeMountDirectory
+        type: string
+        label: "Directory on Node"
+        description: "The directory on the node to use for repository storage"
+        required: true
+        default: "/mnt/repo-storage"
+      - variable: volumes.podMountDirectory
+        type: string
+        label: "Directory on Pod"
+        description: "The directory inside the pod to mount the volume."
+        required: true
+        default: "/var/data"
+
+  - variable: datadog.enabled
+    type: boolean
+    required: true
+    group: "Application"
+    label: "Enable DataDog"
+    description: "Turn DataDog APM on for the pipeline"
+
+  - variable: dns.ndots
+    type: string
+    required: false
+    group: "Application"
+    label: "Number of ndots for DNS configuration"
+    description: "Modifies the number of ndots in /etc/resolv.conf"
+    default: 2
+
+  # Reporting Kafka Health Service Settings
+  - variable: application.webPort
+    type: string
+    required: true
+    group: "Application / Reporting Kafka Health Service"
+    label: Application port binding
+    description: The port binding that the application uses to listen to web requests
+    default: "8080"
+
+  - variable: application.nodePort
```

Comment:
Unfortunately there are a finite set of nodeports and "30680" is already taken by data sink. "30780" is free. Lt's use that. This won't be an issue once we have a load balancer, which @cflanny is working on, and we can stop using nodeports.

Reply:
Changed ports (in 2 files) to 30780